module/zcommon/zfeature_common.c

```diff
@@ -598,6 +598,19 @@ zpool_feature_init(void)
 	zfeature_register(SPA_FEATURE_DRAID,
 	    "org.openzfs:draid", "draid", "Support for distributed spare RAID",
 	    ZFEATURE_FLAG_MOS, ZFEATURE_TYPE_BOOLEAN, NULL);
+
+
+	{
+	static const spa_feature_t xattr_compat_deps[] = {
+		SPA_FEATURE_EXTENSIBLE_DATASET,
```

Comment:
As a little future insurance we should consider depending on `SPA_FEATURE_ENABLED_TXG`.  This way it'll always be possible to determine when the feature was enabled, and if we always write xattrs with the compat format it would be possible to determine what format they're in (even if we normally use a different mechanism for this).